.github/workflows/build.yml

```diff
@@ -754,3 +755,16 @@ jobs:
                 version: '${{ github.event.pull_request.head.sha || github.sha }}'
               }
             })
+
+  notify:
+    if: ${{ always() && contains(needs.*.result, 'failure') && github.ref == 'refs/heads/master' }}
+    runs-on: ubuntu-latest
+    needs:
+      - deploy
+    steps:
+      - name: Report failure
+        uses: espoon-voltti/voltti-actions/notify@master
+        with:
+          webhook_url: ${{ secrets.SLACK_WEBHOOK_URL }}
+          channel: "#evaka-dev-alerts"
+          message: "CI job failed"
```

Comment:
```suggestion
          channel: "#evaka-alerts"
          message: "CI job for master branch failed"
```

Reply:
I think there's not many devs even following the #evaka-dev-alerts channel, I think this is important enough to show on the main alerts channel